README.md

```diff
@@ -176,6 +176,8 @@
     <li><a href="https://github.com/yakivy/poppet">Poppet</a></li>
     <li><a href="https://github.com/vitaliihonta/scala-ql">ScalaQL</a></li>
     <li><a href="https://github.com/VladKopanev/zio-saga">ZIO Saga</a></li>
+    <li><a href="https://github.com/plokhotnyuk/jsoniter-scala">Jsoniter Scala</a></li>
+    <li><a href="https://github.com/plokhotnyuk/rtree2d">RTree2D</a></li>
```

Comment:
```suggestion
    <li><a href="https://github.com/plokhotnyuk/jsoniter-scala">Jsoniter Scala</a></li>
    <li><a href="https://github.com/yakivy/poppet">Poppet</a></li>
    <li><a href="https://github.com/plokhotnyuk/rtree2d">RTree2D</a></li>
    <li><a href="https://github.com/vitaliihonta/scala-ql">ScalaQL</a></li>
    <li><a href="https://github.com/VladKopanev/zio-saga">ZIO Saga</a></li>
```

For now, we want to keep them in alphabetical order. We will rearrange them later when we figure out the best format for presenting this libraries))